ioctl/cmd/account/accountinfo.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/spf13/cobra"
+
+	ioAddress "github.com/iotexproject/iotex-address/address"
```

Comment:
`/iotexproject/iotex-address/` is also considered an external github repo
so group it together with cobra above